src/cGame_logic.cpp

```diff
@@ -1844,6 +1844,16 @@ void cGame::onKeyPressedGamePlaying(const cKeyboardEvent &event)
         }
     }
 
+    if (event.hasKeys(SDL_SCANCODE_LALT,SDL_SCANCODE_RETURN)) {
+        if (m_windowed) {
+            m_Screen->setFullScreenMode();
```

Comment:
awesome that it is now this easy. I was wondering if we can get rid of `m_windowed` as afaik it is used as a 'setting' property?